contributing.md

```diff
@@ -0,0 +1,99 @@
+# Contributing Guidelines
+
+This document will guide you through the process of contributing to our projects and help maintain a healthy and
+collaborative environment.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [Issues](#issues)
+3. [Making Changes](#making-changes)
+   - [Coding Conventions](#coding-conventions)
```

Comment:
want to use 4-space, instead?

Reply:
358127c35f19daf362fdc385fe6d98ca5ec93d6f